libs/adapters/src/redis/RedisCacheAdapter.ts

```diff
@@ -385,14 +382,14 @@ export class RedisCache implements Cache {
     const data = {} as any;
     try {
       if (this.initialized()) {
```

Comment:
Let's reverse all of these `if (this.initialized())` and remove the undefined return types